chance/p5js/chance-praegnanz/index.md

```diff
@@ -1,11 +1,11 @@
 ---  
 layout: p5js
-author: [josues]
-author-url: [https://incom.org/profil/7029]
-title: [Law of Praegnanz and chance]
-tags: [P5.js,[CHANCE], [PROXIMITY], Law of Praegnanz]
-permalink: /[CHANCE]/[chance-praegnanz]/
-summary: [This creates random Praegnanzen between ellipses.]
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of praegnanz and chance
+tags: P5.js, chance, proximity, Law of praegnanz
```

Comment:
Sorry. Tags need the `[]` Take a look at https://raw.githubusercontent.com/fabianmoronzirfas/gestalten-in-code/master/grid/p5js/lines/index.md